### PR TITLE
task page break in different component and add top nav bar

### PR DESCRIPTION
**Pull request by amitAMPL**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,9 +2,17 @@ import { NgModule } from '@angular/core';
...
+import { TaskComponent } from './pages/content/task/task.component';
+import { ContentComponent } from './pages/content/content.component';
+
 const routes: Routes = [
   { path:'tasks', component:TasksComponent },
-  { path:'task-list', component:TaskListComponent},
+  { path:'task-list', component:TaskListComponent },
+  { path:'app-header', component:HeaderComponent },
+  { path:'app-task', component:TaskComponent },
+  { path:'**', component:TasksComponent },
```
Comment (prodeepc) [inline src/app/app-routing.module.ts]: Dont use Wild Card entry

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -2,9 +2,17 @@ import { NgModule } from '@angular/core';
...
+import {HeaderComponent} from './pages/header/header.component';
+import { TaskComponent } from './pages/content/task/task.component';
+import { ContentComponent } from './pages/content/content.component';
+
 const routes: Routes = [
   { path:'tasks', component:TasksComponent },
-  { path:'task-list', component:TaskListComponent},
+  { path:'task-list', component:TaskListComponent },
+  { path:'app-header', component:HeaderComponent },
+  { path:'app-task', component:TaskComponent },
```
Comment (prodeepc) [inline src/app/app-routing.module.ts]: Remove extra routes

File: src/app/app.component.ts [hunk 3]
```diff
@@ -7,4 +8,9 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'client';
+
+  constructor(private defaultRoute: Router) {
+    this.defaultRoute.navigate(['/tasks']);
```
Comment (prodeepc) [inline src/app/app.component.ts]: Don't navigate from app component

File: src/app/pages/content/task/task.component.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<div class="sideMargin">
+  <mat-grid-list cols="1" rowHeight="45px" gutterSize="1px">
+    <!-- <div class="header">
+      <div class="titleHeader">Tasks <button class="dotBtn">...</button></div>
+    </div>
+    <div class="addTaskSection">
+      <button class="plusIcon">+</button> Add a task
+    </div> -->
```
Comment (prodeepc) [inline src/app/pages/content/task/task.component.html]: remove commented code

**Review by prodeep-c — APPROVED** [review]

**Review by acmemindsdev — APPROVED** [review]

**Review by prodeep-acme — APPROVED** [review]

**Pull request merged**